backend/webapi/Features/Credentials/BCProvider.ResetMFA.cs

```diff
@@ -0,0 +1,69 @@
+namespace Pidp.Features.Credentials;
+
+using System.Text.Json.Serialization;
+using DomainResults.Common;
+using FluentValidation;
+using HybridModelBinding;
+using Microsoft.EntityFrameworkCore;
+using Pidp.Data;
+using Pidp.Infrastructure.Auth;
+using Pidp.Infrastructure.HttpClients.BCProvider;
+
+public class BCProviderResetMFA
+{
+    public class Command : ICommand<IDomainResult>
+    {
+        [JsonIgnore]
+        [HybridBindProperty(Source.Route)]
+        public int PartyId { get; set; }
```

Comment:
You only need hybrid model binding when there are parameters coming from multiple sources, i.e. route + body + query string etc. You can drop the hybrid model binding attributes and change the binding to `[FromRoute]` in the controller.

Reply:
We should also drop the `[JsonIgnore]` as well. There are two reasons we put that on our models: 
1. The parameter is directly set by us and we want to ignore anything the FE sends for that parameter (eg. UserId which we pull from the token).
2. To clean up our documentation when doing hybrid model binding. Swagger serializes our models that bind from the body, so the parameter will wrongly appear as a `body` parameter in our swagger if we don't `JsonIgnore` it.